context/config.go

```diff
@@ -61,13 +63,35 @@ type AnkhConfig struct {
 	Contexts                  map[string]Context `yaml:"contexts"`
 }
 
+type KubeCluster struct {
+	Cluster struct {
+		Server string `yaml:"server"`
+	}
+	Name string `yaml:"name"`
+}
+
+type KubeContext struct {
+	Context struct {
```

Comment:
Does this also need a `user` field? The kube-config has that, what does it get used for?